src/gtk/menu.cpp

```diff
@@ -1313,13 +1346,20 @@ static wxString GetGtkHotKey( const wxMenuItem& item )
                 }
                 break;
         }
-
-        delete accel;
     }
 
     return hotkey;
 }
 
+static wxString GetGtkHotKey( const wxMenuItem& item )
```

Comment:
In fact, looking at the code, I'm not even sure if we still need this function, it's used in a single place only and it looks like it might be simpler to just inline it into `wxGetGtkAccel()`